### PR TITLE
Account delete confirmation step modal create

### PR DESCRIPTION
**Pull request by kkomaz**
Closes: https://github.com/blockstack/blockstack-browser/issues/744

Couple of Confirmations needed:
1. Coding style matches overall application.., ie. naming conventions, etc
2. Confirmation Text needed for DeleteAccountModal

Screenshot:
screenshot of google chrome 9-13-17 5-52-46 pm

Demo:
demo

### CODE REVIEW

**Comment by guylepage3** [comment]
@kkomaz this is awesome. I will review and comment now.

File: app/js/account/DeleteAccountPage.js [hunk 1]
```diff
@@ -94,11 +105,17 @@ class DeleteAccountPage extends Component {
             data={this.state} onChange={this.onValueChange}
           />
           <div className="container m-t-40">
-            <button className="btn btn-primary" onClick={this.deleteAccount}>
+            <button className="btn btn-primary" onClick={this.openModal}>
```
[the remark this answers is not in the document]
Comment (kkomaz) [reply app/js/account/DeleteAccountPage.js]: Are you referring to the DeleteAccountPage or DeleteModal component? (confirm and cancel step)

[the remark this answers is not in the document]
Comment (guylepage3) [reply app/js/account/DeleteAccountPage.js]: I'm referring to `className="btn btn-primary [btn-block]"`

[the remark this answers is not in the document]
Comment (guylepage3) [reply app/js/account/DeleteAccountPage.js]: My bad.. this should be on the modal. will comment there.

File: app/js/account/components/DeleteAccountModal.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      <p>
+        This will delete your account information from Blockstack.
+        Please save your backup phrase and password
+        if you want to retrieve your account in the future.
+      </p>
+    </div>
+    <div className="delete-account-buttons">
+      <Link
+        onClick={props.closeModal}
+        className="btn btn-secondary btn-sm"
```
Comment (guylepage3) [inline app/js/account/components/DeleteAccountModal.js]: I'm referring to `className="btn btn-primary [btn-block]"` here on line 26

File: app/js/account/components/DeleteAccountModal.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      >
+        Cancel
+      </Link>
+      <Link
+        className="btn btn-danger btn-sm"
```
Comment (guylepage3) [inline app/js/account/components/DeleteAccountModal.js]: and here.. `className="btn btn-primary [btn-block]"` here on line 31

**Comment by kkomaz** [comment]
@guylepage3 

1. Buttons are now displayed block.
2. Change text from delete to remove

screenshot of google chrome 9-14-17 4-29-51 pm

**Comment by guylepage3** [comment]
Great work here @kkomaz! Will pull this in this weekend. 😄

**Pull request merged**